tests/env/test_create.py

```diff
@@ -105,25 +105,59 @@ def test_create_advanced_pip(
     tmp_envs_dir: Path,
     support_file_isolated,
```

Comment:
I suggest adding `tmp_path` here, copy the two argh files into `tmp_path`, do git, and leave the cleanup to pytest.
Or did we already copy those files in the support_file_isolated fixture? leaving only the git calls to this test.